### PR TITLE
De22327 null values

### PR DESCRIPTION
**Pull request by dme722**
Numpy NaN values were not being properly set to `Null` in records.

### CODE REVIEW

**Comment by AshtonUPS** [comment]
<a href="https://imgflip.com/i/3h083b"></a>

File: snakeplane/constants.py [hunk 1]
```diff
@@ -0,0 +1 @@
+SNAKEPLANE_NULL_VALUE_PLACEHOLDER = "sNaKePlAnE_nUlL_vAlUe"
```
Comment (Ninjef) [inline snakeplane/constants.py]: Could also just use a UUID, here's a V4 UUID from [here](https://www.uuidgenerator.net/version4):
`0c2856d0-f631-445a-a112-33fa282cb5de`

Comment (Ninjef) [reply snakeplane/constants.py]: Not a big deal either way though.

Comment (johnkabler) [reply snakeplane/constants.py]: we'd want this to be constant, and not change.  I'm ok with a UUID as long as it's a constant, and not randomly generated each time.

Comment (Ninjef) [reply snakeplane/constants.py]: Yes that's what I mean. Wasn't clear though sorry.

Comment (dme722) [reply snakeplane/constants.py]: You guys are so lame.

Comment (AshtonUPS) [reply snakeplane/constants.py]: <a href="https://imgflip.com/i/3h9efv"></a>

Comment (Ninjef) [reply snakeplane/constants.py]: image

**Pull request merged**